board/freescale/mx6ullevk/mx6ullevk_siklu_ethernity.c

```diff
@@ -23,9 +23,21 @@ static int do_siklu_ethernity_reset(cmd_tbl_t * cmdtp, int flag, int argc,
 	int rc = 0;
 	T_CPLD_LOGIC_WD_RW_REGS reset_reg;
 
+	/* Drive to 0V */
 	reset_reg.uint8 = siklu_cpld_read(R_CPLD_LOGIC_WD_RW);
 	reset_reg.s.cfg_fpga_8020_rst = 1;
 
+	rc = siklu_cpld_write(R_CPLD_LOGIC_WD_RW, reset_reg.uint8);
+	if (rc != 0) {
+		printf("%s() ERROR writing to CPLD, line %d\n", __func__, __LINE__);
+	}
+
+	mdelay(20);
+
+	/* Release to 3.3V pullup */
+	reset_reg.uint8 = siklu_cpld_read(R_CPLD_LOGIC_WD_RW);
```

Comment:
Is this read necessary? Is there any reason to believe that read value is different from the same read above (except from `cfg_fpga_8020_rst`)?

Reply:
I don't know. I did it because I though that somehow it could be changed by some other action in the 20 ms interval that passed.

Reply:
If `R_CPLD_LOGIC_WD_RW` can change behind our back, it can also change between read and write. SPI transactions take a few µs at least.

Reply:
Ok, will remove.